demo/admin/src/config.tsx

```diff
@@ -20,28 +18,9 @@ export function createConfig() {
         ...cometConfig,
         apiUrl: environmentVariables.API_URL,
         adminUrl: environmentVariables.ADMIN_URL,
-        sitesConfig: JSON.parse(atob(environmentVariables.PUBLIC_SITE_CONFIGS)) as PublicSiteConfig[],
+        siteConfigs: JSON.parse(atob(environmentVariables.PUBLIC_SITE_CONFIGS)) as PublicSiteConfig[],
         buildDate: environmentVariables.BUILD_DATE,
         buildNumber: environmentVariables.BUILD_NUMBER,
         commitSha: environmentVariables.COMMIT_SHA,
     };
 }
-
-type Config = ReturnType<typeof createConfig>;
-
-const ConfigContext = createContext<Config | undefined>(undefined);
-
-export function ConfigProvider({ config, children }: PropsWithChildren<{ config: Config }>) {
-    return <ConfigContext.Provider value={config}>{children}</ConfigContext.Provider>;
-}
-
-/** @knipignore */
-export function useConfig(): Config {
```

Comment:
this application specific config you removed now. Don't we need it?

Reply:
We didn't use it anywhere (hence the `@knipignore`). However it is still possible to provide custom values using module augmention:

```ts
declare module "@comet/cms-admin" {
    interface CometConfig {
        demo: boolean;
    }
}
```

<img width="630" alt="image" src="https://github.com/user-attachments/assets/e0ce1326-48f9-42d2-8c8b-a212bc61e6d7" />

Is this sufficient?

Reply:
Removed: 4ed40aca66dfe86032877e9910d1bd0ac35b5b69